instrumentation/sequel/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History: opentelemetry-instrumentation-sequel
+
+### v0.1.0 / 2023-09-30
+
+* Initial release
```

Comment:
The release automation generates the changelog entries for you. It requires this file be empty for the initial release. 

```suggestion
```